app/package.json

```diff
@@ -7,8 +7,8 @@
     "dev": "next dev",
     "format": "prettier --write '**/*.{js,json,md,mdx,ts,tsx,scss,yaml,yml}'",
     "format-check": "prettier --check '**/*.{js,json,md,mdx,ts,tsx,scss,yaml,yml}'",
-    "lint": "next lint",
-    "lint-fix": "next lint --fix",
+    "lint": "next lint --dir src --dir stories --dir .storybook --dir tests --dir scripts --dir app --dir lib --dir types",
```

Comment:
ℹ️ Discovered that `next lint` only lints `components/`, `lib/`, and `src/` directories by default, so our CI wasn't linting test or Storybook files